tests/testapp/tests/test_serialization.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.test import TestCase
+from facility_profile.models import Facility
+from morango.utils.syncing_utils import _syncing_models
+
+
+class SerializationTestCase(TestCase):
+
+    def setUp(self):
+        self.bob = Facility.objects.create(name="bob")
+        self.dob = Facility.objects.create(name="dob")
+        self.bob.id = self.bob.id.hex
+        self.dob.id = self.dob.id.hex
+        self.bob_dict = self.bob.serialize()
+        self.dob_dict = self.dob.serialize()
+
+    def test_serialization(self):
+        self.assertEqual(self.bob_dict['name'], 'bob')
+        self.assertEqual(self.bob.morango_model_name, Facility.morango_model_name)
+
+    def test_field_deserialization(self):
+        class_model = _syncing_models[self.bob.morango_model_name]
+        self.bob_copy = class_model.deserialize(self.bob_dict)
+        for f in Facility._meta.concrete_fields:
+            if isinstance(f, models.DateTimeField):
+                continue
```

Comment:
What I realized is that through serialization and deserialization of this field, we lose 3 digits of precision, so they do not come out equal. I could probably rewrite the test to account for this.